packages/nextjs/tsconfig.json

```diff
@@ -9,13 +9,16 @@
     "noEmit": true,
     "esModuleInterop": true,
     "module": "esnext",
-    "moduleResolution": "node",
+    "moduleResolution": "Bundler",
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
     "incremental": true,
     "paths": {
-      "~~/*": ["./*"]
+      "~~/*": ["./*"],
+      "abitype": ["./node_modules/abitype"],
+      "viem/node_modules/abitype": ["./node_modules/viem/node_modules/abitype"],
+      "wagmi/node_modules/abitype": ["./node_modules/wagmi/node_modules/abitype"]
```

Comment:
I think we can remove viem and wagmi mapping, from here now?

Reply:
Thanks! [6023ccb](https://github.com/scaffold-eth/scaffold-eth-2/pull/885/commits/6023ccb96d0371acc01ccb5945980992ca999441)